runopts.h

```diff
@@ -130,6 +130,11 @@ typedef struct svr_runopts {
 	char * allowed_pw;
 #endif
```

Comment:
please fix this #ifdef too (should be #if)

Reply:
Done.